eth2util/signing/signing_test.go

```diff
@@ -263,12 +264,35 @@ func TestVerifyAggregateAndProof(t *testing.T) {
 	require.NoError(t, err)
 
 	sig, pubkey := sign(t, sigData[:])
-	agg.Signature = tblsconv.SigToETH2(sig)
+	agg.Signature = sig
 
 	require.NoError(t, signing.VerifyAggregateAndProof(context.Background(), bmock, pubkey, agg))
 }
 
-func sign(t *testing.T, data []byte) (*bls_sig.Signature, *bls_sig.PublicKey) {
+func TestVerifySyncCommitteeMessage(t *testing.T) {
```

Comment:
can we make this a table test rather?

Reply:
since each test require different signing data and calls different function, i don't think it would add much value converting these tests to table test.